test/Concurrency/predates_concurrency_import.swift

```diff
@@ -0,0 +1,24 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -emit-module -emit-module-path %t/StrictModule.swiftmodule -module-name StrictModule -warn-concurrency %S/Inputs/StrictModule.swift
+// RUN: %target-swift-frontend -emit-module -emit-module-path %t/NonStrictModule.swiftmodule -module-name NonStrictModule %S/Inputs/NonStrictModule.swift
+// RUN: %target-swift-frontend -emit-module -emit-module-path %t/OtherActors.swiftmodule -module-name OtherActors %S/Inputs/OtherActors.swift -disable-availability-checking
+
+// RUN: %target-typecheck-verify-swift -typecheck  -I %t
+
+@_predatesConcurrency import NonStrictModule
+@_predatesConcurrency import StrictModule
+@_predatesConcurrency import OtherActors
+// expected-remark@-1{{'@_predatesConcurrency' attribute on module 'OtherActors' is unused}}{{1-23=}}
```

Comment:
do remarks actually show up in command line too? I don't think I ever saw one 🤔

Reply:
Yes, they do

Reply:
Great, thanks :)